src/main/java/pl/edu/mimuw/chess/ChessBoard.java

```diff
@@ -0,0 +1,129 @@
+package pl.edu.mimuw.chess;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.Random;
+
+import pl.edu.mimuw.chess.Util;
+
+public class ChessBoard {
+	public static final int BOARD_SIZE = 8;
+	
+	public static final String EMPTY_BOARD = "╔═╤═╤═╤═╤═╤═╤═╤═╗\n" +
```

Comment:
warto wiedzieć jako coś 😉 https://blogs.oracle.com/javamagazine/post/text-blocks-come-to-java